.github/workflows/test_mlagility.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         shell: bash -el {0}
         run: |
           pylint src/mlagility --rcfile .pylintrc
-          pylint examples --rcfile .pylintrc --ignore-paths examples/onnxflow
+          pylint examples --rcfile .pylintrc --ignore-paths examples/onnxflow --disable E0401,E0611
```

Comment:
Isn't it a better solution to remove `import-error` and `no-name-in-module` from `.pylintrc`?

Reply:
pylintrc is global for the whole project. I only disabled those errors for this specific folder.